skimage/io/_plugins/tifffile_plugin.py

```diff
@@ -1,9 +1,6 @@
 from warnings import warn
 
-try:
-    from tifffile import TiffFile, imsave, parse_kwargs
-except ImportError:
-    from ...external.tifffile import TiffFile, imsave, parse_kwargs
+from tifffile import TiffFile, imsave, parse_kwargs
 
 
 def imread(fname, dtype=None, **kwargs):
```

Comment:
There might be some behaviour here that we want to deprecate. We can leave that for an other PR though.